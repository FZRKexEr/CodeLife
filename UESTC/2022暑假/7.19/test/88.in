10
djhvtdguzgsqqgzlmoybnhtbyulkohaspfbvpsupgoeixflrxifmzdminzvzhbtqfxdyxfivrvkfktuzpaeycprvzxuqztzhoomb
2
rpvamjcweszzugseedsaphukftabfyhrwdpxkodudenpzbfwnviqjivpqwcupgkjtqbumprhscwwvehdssuqcxqvtmksfomftgju
6
kbuliabfkqjfvenlggxxtlhckkwimkviadvmhbzhmtlggyxxzzoytrbyhfbwgartwvhtuuftatewjvtbquigrvcefmmhsbexbggs
10
eyrvppqxumkvnvnejrswhzdrplseqoknavxaupgzrkignvldjvnhwuxqyfscyjqzgwynblifaoyuasueaomgpsfgvhewqeusytnj
6
kxqitvlqahcllqctqdnvfycbndnmvrugyormjpbmwynnaekpzxgcgopjipugfssgwsysdbcpfwyluesdzjjupjqarwzxkzqtmlcc
9
kttclzgfxvhbkgatciqdmylitoniiylzurjppmciuqggpyyelwixspobgmzhopcxucoinxihpbaqnjstzsoyxefhudajktbhazuf
8
rskvmfegkabficqeuaewztmosoajuqrzpedkbqwoeiiaumdmpecmwmxhqdauqilefkwazrnplwpjpdyzcwkpvjwceebacoylmblq
2
vtwtmihbcllfkmfuuabzdbedvceexvkadcmhspabveyhcdqnsbpmflfgcqlcgvfcgfkevgpmdkjsyegsxingawrcdsmzpljmlldw
3
nvmcgskhmxoyfyvuqmyvaxeiwzloptuajmmzlcdmamaxthkgpaqjseakuxmeothukvypqyoudtknnbeskiycxkgdblgjuctwcrqg
1
joppyereniigocknvojjhmdlnicpdqqxtigopwyrpalkrxuufmkfatsashlfbdwrathqpudzeoqpcmieavoptgfipjjulagbranl
10
