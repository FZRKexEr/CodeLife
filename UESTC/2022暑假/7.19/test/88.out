65980984
65536
1024
65536
2048
4096
65980984
603979768
882499718
1024
